### PR TITLE
Add SetStatus to getting started

### PR DESCRIPTION
**Pull request by cijothomas**
Now that there is proper support for `Status`, I think this concept is worth documenting in getting started. Open to other suggestions as well.

### CODE REVIEW

File: docs/trace/getting-started/Program.cs [hunk 1]
```diff
@@ -36,5 +36,6 @@ public static void Main()
         activity?.SetTag("foo", 1);
         activity?.SetTag("bar", "Hello, World!");
         activity?.SetTag("baz", new int[] { 1, 2, 3 });
+        activity?.SetStatus(ActivityStatusCode.Ok);
```
Comment (cijothomas) [inline docs/trace/getting-started/Program.cs:39]: depending on other's opinion on this, I will explore documenting this somewhere else...

Comment (reyang) [reply docs/trace/getting-started/Program.cs:39]: maybe Error is better for demo purpose?

**Review by reyang — APPROVED** [review]

File: docs/trace/getting-started/Program.cs (hunk 1 above)
Comment (cijothomas) [reply docs/trace/getting-started/Program.cs:39]: I thought about it.. But it'd be more apt to show Error, with an Exception.. that would then bring all the complexities associated with Exception into a getting-started doc..... (we already have a section on Exception-reporting, which I'll tweak to reflect new Status api)

Comment (reyang) [reply docs/trace/getting-started/Program.cs:39]: Makes sense, thanks!

**Review by Yun-Ting — APPROVED** [review]

**Pull request merged**